aspnetcore/host-and-deploy/iis/modules.md

```diff
@@ -15,75 +15,75 @@ uid: host-and-deploy/iis/modules
+| Module | .NET Core Active | ASP.NET Core Option |
+| ------ | :--------------: | ------------------- |
+| **Anonymous Authentication**<br>`AnonymousAuthenticationModule` | Yes | |
+| **Basic Authentication**<br>`BasicAuthenticationModule` | Yes | |
+| **Client Certification Mapping Authentication**<br>`CertificateMappingAuthenticationModule` | Yes | |
+| **CGI**<br>`CgiModule` | No | |
+| **Configuration Validation**<br>`ConfigurationValidationModule` | Yes | |
+| **HTTP Errors**<br>`CustomErrorModule` | No | [Status Code Pages Middleware](xref:fundamentals/error-handling#configuring-status-code-pages) |
+| **Custom Logging**<br>`CustomLoggingModule` | Yes | |
+| **Default Document**<br>`DefaultDocumentModule` | No | [Default Files Middleware](xref:fundamentals/static-files#serve-a-default-document) |
+| **Digest Authentication**<br>`DigestAuthenticationModule` | Yes | |
+| **Directory Browsing**<br>`DirectoryListingModule` | No | [Directory Browsing Middleware](xref:fundamentals/static-files#enable-directory-browsing) |
+| **Dynamic Compression**<br>`DynamicCompressionModule` | Yes | [Response Compression Middleware](xref:performance/response-compression) |
+| **Tracing**<br>`FailedRequestsTracingModule` | Yes | [ASP.NET Core Logging](xref:fundamentals/logging/index#the-tracesource-provider) |
+| **File Caching**<br>`FileCacheModule` | No | [Response Caching Middleware](xref:performance/caching/middleware) |
+| **HTTP Caching**<br>`HttpCacheModule` | No | [Response Caching Middleware](xref:performance/caching/middleware) |
+| **HTTP Logging**<br>`HttpLoggingModule` | Yes | [ASP.NET Core Logging](xref:fundamentals/logging/index)<br>Implementations: [elmah.io](https://github.com/elmahio/Elmah.Io.Extensions.Logging), [Loggr](https://github.com/imobile3/Loggr.Extensions.Logging), [NLog](https://github.com/NLog/NLog.Extensions.Logging), [Serilog](https://github.com/serilog/serilog-extensions-logging)
+| **HTTP Redirection**<br>`HttpRedirectionModule` | Yes | [URL Rewriting Middleware](xref:fundamentals/url-rewriting) |
+| **IIS Client Certificate Mapping Authentication**<br>`IISCertificateMappingAuthenticationModule` | Yes | |
+| **IP and Domain Restrictions**<br>`IpRestrictionModule` | Yes | |
+| **ISAPI Filters**<br>`IsapiFilterModule` | Yes | [Middleware](xref:fundamentals/middleware/index) |
+| **ISAPI**<br>`IsapiModule` | Yes | [Middleware](xref:fundamentals/middleware/index) |
+| **Protocol Support**<br>`ProtocolSupportModule` | Yes | |
+| **Request Filtering**<br>`RequestFilteringModule` | Yes | [URL Rewriting Middleware `IRule`](xref:fundamentals/url-rewriting#irule-based-rule) |
+| **Request Monitor**<br>`RequestMonitorModule` | Yes | |
+| **URL Rewriting**<br>`RewriteModule` | Yes&#8224; | [URL Rewriting Middleware](xref:fundamentals/url-rewriting) |
+| **Server Side Includes**<br>`ServerSideIncludeModule` | No | |
```

Comment:
Add a hyphen between "Server" and "Side"

Reply:
The funny/sad thing is that MS is inconsistent about it. I think the dashed version edges out the version without a dash looking at MS blogs and MSDN (we're not helping tho with https://docs.microsoft.com/iis/configuration/system.webserver/serversideinclude, where the title is *Server Side Include \<serverSideInclude>*.

The OS isn't all that helpful. Although WinOS has it with a dash ...

![capture1](https://user-images.githubusercontent.com/1622880/36320002-4d51a014-130a-11e8-850b-5ca86647d280.PNG)

WS 2016 doesn't ...

![capture2](https://user-images.githubusercontent.com/1622880/36320012-5492c95c-130a-11e8-8f79-6893f5bcae55.PNG)

😭 ...... Let's go with the dash.